altsrc.go

```diff
@@ -6,6 +6,8 @@ import (
 	"os"
 	"runtime"
 	"strings"
+
+	"github.com/urfave/cli/v3"
```

Comment:
Is it possible to avoid this import? It seems needed only for `cli.ValueSourceChain` interface.

Reply:
Then user would need to convert to cli.ValueSourceChain. I'd like to remove it but dont see any real value in having the user do the conversion